doc/content/concepts/devicetemplates/converting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Converting Templates"
+description: ""
+weight: 3
+---
+
+You can convert data to device templates using formats supported by TTN Stack. Start with listing the supported formats:
```

Comment:
> data

Be more clear about what kind of "data". People read from top to bottom, so maybe a little introduction about a typical process would help here.